Provenance-Emu/Provenance
# more tvOS UX

### What does this PR do
* fix focus issues on `tvOS`
    - for example when you toggle an option the focus would jump to the top, not any more. 
* added a class `TVPopupController` to present another `UIViewController` in a more "popup like way"
    - used this class to show, the now focus fixed, `Library Options`
* put a tittle on each page of the `Settings` screen, and fixed focus issues via `QuickTableViewController`
* this mostly is a `tvOS` only change, but the `iOS` code path was changed so `Library Options` could work on both `iOS` and `tvOS`
    - as a side effect the `Library Options` does not flash when selected on iOS.

### Where should the reviewer start

### How should this be manually tested
* run on iOS on iPad and iPhone and make sure `Library Options` presents and works ok.
    - `Library Options` presents in a popup on iPad.
* run on tvOS and run `Library Options` and `Settings`
    - or just ask MrJs to do it.

### Any background context you want to provide

### What are the relevant tickets

### Screenshots (important for UI changes)

![Simulator Screen Shot - Apple TV - 2022-01-23 at 17 55 48](https://user-images.githubusercontent.com/4494698/150710294-25929b01-62d9-4d7b-9810-ff4483f1225b.png)

![Simulator Screen Shot - Apple TV - 2022-01-23 at 17 55 36](https://user-images.githubusercontent.com/4494698/150710317-6d7a86aa-c7dd-4189-b046-171a4f3a8114.png)

### Questions
